plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -104,17 +104,23 @@ def __init__(self, repo, publish_conduit, config, distributor_type, **kwargs):
         :param distributor_type: The type of the distributor that is being published
         :type distributor_type: str
         """
-        super(ExportRepoPublisher, self).__init__(repo, publish_conduit, config, distributor_type,
-                                                  **kwargs)
 
         date_q = export_utils.create_date_range_filter(config)
-        if date_q:
-            # Since this is a partial export we don't generate metadata
-            # we have to clear out the previously added steps
-            # we only need special version s of the rpm, drpm, and errata steps
-            self.clear_children()
-            self.add_child(PublishRpmAndDrpmStepIncremental(repo_content_unit_q=date_q))
-            self.add_child(PublishErrataStepIncremental(repo_content_unit_q=date_q))
+
+        if config.get_boolean(constants.INCREMENTAL_EXPORT_REPOMD_KEYWORD):
+            super(ExportRepoPublisher, self).__init__(repo, publish_conduit, config,
+                                                      distributor_type, association_filters=date_q,
+                                                      **kwargs)
+        else:
```

Comment:
OK, I think it's reasonable to consider that a problem for later since someone is going to have to fix those tests eventually anyway.